docs/wordpress.md

```diff
@@ -1,4 +1,4 @@
-# Installing Wordpress with Carrier
```

Comment:
I think we should remove this file. It's no longer accurate (or is it ?).

Reply:
yep, I planned to do in next PR, it does not really belong to 'renaming' topic

Reply:
there are few other things to remove

Reply:
ok